src/chericat.c

```diff
@@ -83,19 +83,6 @@ main(int argc, char *argv[])
 	// the program arguments would then be handled as intended without the libxo arguments.
 	argc = xo_parse_args(argc, argv);
 
-
```

Comment:
You can remove the above line modifying `argc` and comments above it as it's not needed any more. Also, you can remove from this file:
```
#include <libxo/xo.h>
```
and from `Makefile`:
```
-lxo
```


I wish GitHub allowed to comment on lines that were not modified in a pull request. I can only describe these issues here.

Reply:
We still need the libxo, the function is there for outputting in libxo format on the console. I removed the temp file path which was intended for outputing to a file instead of the console.

Reply:
Oh, ok. Thanks for clarifying!